### DISCUSSION TITLE
Formulario

### DISCUSSION THREAD

**Discussion by CrispisMAB** [LaunchX-InnovaccionVirtual/CursoIntroPython#338] [Q&A]
Hola buenos días, ya tengo mis katas listas para contestar el formulario sin embrago desde ayer en la noche no aparecía habilitado :(, alguien mas esta en la misma situación? o sabrán de que forma solucionarlo?
![imagen_2022-02-21_113011](https://user-images.githubusercontent.com/84803587/155003022-f7e31edd-7fca-4c19-832c-339a48a31384.png)

**Comment by doguedogue** [comment]
Tienes que ingresar con tu cuenta de @innovaccion y usa un navegador de incognito, por lo de las sesiones

**Comment by DavidHdz1997** [comment]
Me pasa lo mismo, no me deja desde ningun navegador y segun esto la fecha limite era hoy hasta las 8pm, como le hacemos?

**Comment by ArielCanedo** [comment]
Reintenta, prueba en incógnito, sino intenta mandar un mensaje por insta
